packages/block-directory/src/components/downloadable-block-header/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { BlockIcon } from '@wordpress/block-editor';
+import BlockRatings from '../block-ratings';
+
+function DownloadableBlockHeader( { icon, title, rating, ratingCount, onClick } ) {
+	return (
+		<div className="block-directory-downloadable-block-header__row">
+			{
+				icon.match( /\.(jpeg|jpg|gif|png)/ ) !== null ?
+					<img src={ icon } alt="block icon" /> :
```

Comment:
Is there something more descriptive than `"block icon"` that we can use? Maybe the block's title? Regardless, this string needs to be localised.